external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/builders/SingleTopicKafkaSpoutConfiguration.java

```diff
@@ -21,15 +21,10 @@
 import static org.apache.storm.kafka.spout.KafkaSpoutConfig.FirstPollOffsetStrategy.EARLIEST;
 
 import org.apache.kafka.clients.consumer.ConsumerConfig;
```

Comment:
I would call package where this class lives `config.builder` instead of `builders`, which is a bit misleading since this is really a configuration class.